tests/print_uniform-f16.ispc

```diff
@@ -0,0 +1,11 @@
+// rule: skip on arch=x86
+// rule: skip on arch=x86_64
+// rule: skip on arch=x86-64
+export void print_uf(uniform float a) {
+    uniform float16  uniFloatVal   =  (float16)a + 4.75f16;
+    print("Test uniform float: %\n", uniFloatVal);
```

Comment:
Could you please run `clang-format`  on the new tests?

Reply:
Done